### PR TITLE
Remove old admin interface 

### PR DESCRIPTION
**Pull request by lkiesow**
This patch removes the old admin interface frontend.

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://docs.opencast.org/develop/developer/#participate/development-process/#automatically-closing-issues-when-a-pr-is-merged) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: etc/org.opencastproject.organization-mh_default_org.cfg [hunk 1]
```diff
@@ -167,210 +167,3 @@ prop.org.opencastproject.admin.mediamodule.url=${prop.org.opencastproject.engage
...
-# - optional: Like `never` but present users with a button in the series permission dialog which allows them to
-#             replace the event specific permissions if they want to.
-# Default: optional
-#prop.admin.series.acl.event.update.mode=optional
-
-# Whether the ACL of a new event is initialized with the ACL of its series.
-#
-# Format: Boolean
-# Default: true
-#prop.admin.init.event.acl.with.series.acl=true
```
Comment (mtneug) [inline etc/org.opencastproject.organization-mh_default_org.cfg]: This property is still used by the new Admin UI.

File: etc/org.opencastproject.organization-mh_default_org.cfg [hunk 2]
```diff
@@ -167,210 +167,3 @@ prop.org.opencastproject.admin.mediamodule.url=${prop.org.opencastproject.engage
...
-
-# Enable themes in the admin interface.
-# Format: boolean
-# Default: true
-prop.admin.themes.enabled=false
-
-# Enable the sttatistics views in the admin intterface.
-# Format: boolean
-# Default: false
-#prop.admin.statistics.enabled=false
```
Comment (mtneug) [inline etc/org.opencastproject.organization-mh_default_org.cfg]: This property is still used by the new Admin UI.

File: etc/org.opencastproject.organization-mh_default_org.cfg [hunk 3]
```diff
@@ -167,210 +167,3 @@ prop.org.opencastproject.admin.mediamodule.url=${prop.org.opencastproject.engage
...
-# Common values:
-#   The internal editor: #!/events/events/$id/tools/editor
-#   The external editing tool: /editor-ui/index.html?id=$id
-# Default: /editor-ui/index.html?id=$id
-#prop.admin.editor.url=/editor-ui/index.html?id=$id
-
-# Enable themes in the admin interface.
-# Format: boolean
-# Default: true
-prop.admin.themes.enabled=false
```
Comment (mtneug) [inline etc/org.opencastproject.organization-mh_default_org.cfg]: This property is still used by the new Admin UI.

**Review by mtneug — DISMISSED** [review]

File: etc/org.opencastproject.organization-mh_default_org.cfg (hunk 1 above)
Comment (lkiesow) [reply etc/org.opencastproject.organization-mh_default_org.cfg]: You are right. That has been recently added, but was never moved from the deprecated section in the Opencast configs.

**Comment by lkiesow** [comment]
Waiting for #5947 and #5976 before rebasing

**Comment by lkiesow** [comment]
Actually, we also need to have #5946 merged before I can rebase this

**Comment by gregorydlogan** [comment]
@lkiesow 5946 is merged, let's get this done.

**Review by gregorydlogan — CHANGES_REQUESTED** [review]
I think you got most of it, but I found a few missed references:
- [x] `docs/guides/developer/docs/develop/frontends/overview.md` references the old and new admin UIs.
- [x] `docs/guides/developer/docs/develop/frontends/admin-ui/development.md` contains an (out of date?) style guide, and development instructions for the old admin UI.
- [x] `docs/guides/developer/docs/participate/localization.md` contains a reference `admin-ui-frontend/app/img/lang` for adding SVG flags.
- [x] `docs/guides/admin/docs/configuration/admin-ui/asset-upload.md` references a bunch of localization related files in `admin-ui-frontend`.

**Pull request merged**